python/ray/data/_internal/execution/interfaces/op_runtime_metrics.py

```diff
@@ -108,9 +109,13 @@ class OpRuntimeMetrics:
     # === Miscellaneous metrics ===
 
     # Time spent generating blocks in tasks.
-    block_generation_time: float = field(
-        default=0, metadata={"map_only": True, "export_metric": True}
-    )
+    block_generation_time: float = field(default=0, metadata={"map_only": True})
```

Comment:
should we keep `"export_metric": True` for this?

Reply:
Oh yeah, lemme change back!